### PR TITLE
Match latest schema

### PR DESCRIPTION
**Pull request by anderspitman**

### CODE REVIEW

**Comment by anderspitman** [comment]
No need for a full review yet. There will be more changes to get dfaker up to date with the current upload schema.

File: dfaker/wizard.py [hunk 1]
```diff
@@ -1,85 +1,117 @@
...
-                                                    - tools.round_to(wizard_reading["insulinOnBoard"]))
+            carb = (wizard_reading["carbInput"] /
+                    wizard_reading["insulinCarbRatio"])
+            wizard_reading["recommended"]["carb"] = tools.round_to(carb)
+            wizard_reading["recommended"]["correction"] = 0
+            corrected_carb = (wizard_reading["recommended"]["carb"] +
+                              wizard_reading["recommended"]["correction"])
+            iob = tools.round_to(wizard_reading["insulinOnBoard"])
+            wizard_reading["recommended"]["net"] = (corrected_carb - iob)
+            wizard_reading["units"] = "mmol/L"
```
Comment (anderspitman) [inline dfaker/wizard.py:80]: This is the only real change. Everything else should be refactoring for linting and clarity.

File: dfaker/common_fields.py [hunk 2]
```diff
@@ -4,19 +4,25 @@
...
\ No newline at end of file
+    local_datetime = datetime.fromtimestamp(timestamp)
+    datatype["timezoneOffset"] = tools.get_offset(zonename, local_datetime)
+    offset_time_seconds = datatype["timezoneOffset"] * 60
+    offset_time_struct_utc = time.gmtime(timestamp + offset_time_seconds)
+    datatype["deviceTime"] = time.strftime('%Y-%m-%dT%H:%M:%S',
+                                           offset_time_struct_utc)
+    time_struct_utc = time.gmtime(timestamp)
+    datatype["time"] = time.strftime('%Y-%m-%dT%H:%M:%S.000Z', time_struct_utc)
+    datatype["conversionOffset"] = 0
```
Comment (anderspitman) [inline dfaker/common_fields.py:27]: This is the only real change. Everything else should be refactoring for linting and clarity.

**Comment by anderspitman** [comment]
Hm the refactoring is really obscuring the actual changes. I think I'm going to hold off on that for the remainder of this PR. I marked the most important line changes with comments to hopefully make reviewing easier.

**Comment by jh-bate** [comment]
LGTM - merging

**Pull request merged**